crypto-ffi/Makefile.toml

```diff
@@ -72,14 +72,14 @@ args = [
     "--bin", "uniffi-bindgen",
     "generate",
     "--language", "swift",
-    "--out-dir", "./bindings/swift/WireCoreCrypto/WireCoreCrypto",
+    "--out-dir", "./bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi",
     "--library", "../target/release/deps/libcore_crypto_ffi.${LIBRARY_EXTENSION}"
 ]
 
 [tasks.bindings-swift]
 dependencies = ["_bindings-swift"]
 script = '''
-    mv ./bindings/swift/WireCoreCrypto/WireCoreCrypto/core_crypto_ffi.swift ./bindings/swift/WireCoreCrypto/WireCoreCrypto/CoreCrypto.swift
+    mv ./bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_crypto_ffi.swift ./bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/CoreCrypto.swift
```

Comment:
Can we get rid of this `mv` step entirely by tweaking swift settings in `uniffi.toml`? Maybe changing `cdylib_name` is enough?

Reply:
I'll maybe we can just remove it now that this is an internal library.